homeassistant/components/unifiprotect/views.py

```diff
@@ -59,6 +59,17 @@ def async_generate_event_video_url(event: Event) -> str:
     )
 
 
+@callback
+def async_generate_proxy_event_video_url(
+    nvr_id: str,
+    event_id: str,
+) -> str:
+    """Generate proxy URL for event video."""
+
+    url_format = VideoEventProxyView.url or "{nvr_id}/{event_id}"
```

Comment:
How can `VideoEventProxyView.url` be falsey that the right side would ever evaluate?

Reply:
It can't, its just typed that way.